pyproject.toml

```diff
@@ -19,7 +19,7 @@ dependencies = [
     "fitter~=1.4",
     "lhsmdu~=1.1",
     "networkx~=2.6",
-    "numpy~=1.22",
+    "numpy~=1.22.0",
```

Comment:
Unfortunately the latest `1.24.2` release has depreciated `numpy.int`, which is used by multiple of the other packages this project depends on. 

So we can't pin it to the minor versions as there are breaking changes in their minor patches. So this will allow `1.22.X` instead.